Library/Formula/bibtex2html.rb

```diff
@@ -12,17 +10,34 @@ class Bibtex2html < Formula
     sha1 "32377bea1f584fedf5d2abb604a1d46e5e92ac5c" => :mountain_lion
   end
 
-  depends_on 'objective-caml'
-  depends_on 'hevea'
+  depends_on "objective-caml"
+  depends_on "hevea"
+  depends_on :tex => :optional
 
   def install
     # See: https://trac.macports.org/ticket/26724
-    inreplace 'Makefile.in' do |s|
-      s.remove_make_var! 'STRLIB'
+    inreplace "Makefile.in" do |s|
+      s.remove_make_var! "STRLIB"
     end
 
     system "./configure", "--prefix=#{prefix}"
     system "make"
-    system "make install"
+    system "make", "install"
+  end
+
+  test do
+    (testpath/"test.bib").write <<-EOS.undent
+      @article{Homebrew,
+          title   = {Something},
+          author  = {Someone},
+          journal = {Something},
+          volume  = {1},
+          number  = {2},
+          pages   = {3--4}
+      }
+    EOS
+    system "#{bin}/bib2bib", "test.bib", "--remove", "pages", "-ob", "out.bib"
+    assert File.read("test.bib") =~ /pages\s*=\s*{3--4}/
+    assert File.read("out.bib") !=~ /pages\s*=\s*{3--4}/
```

Comment:
I hit a strange error when using `assert_not_match`, I can't even go to debug mode.

The code:

```
    assert_match /pages\s*=\s*{3--4}/, File.read("test.bib")
    assert_not_match /pages\s*=\s*{3--4}/, File.read("out.bib")
```

And the log:
```
$ brew test ./Library/Formula/bibtex2html.rb -vd
/usr/local/Library/brew.rb (Formulary::FromPathLoader): loading /Users/xucheng/Coding/Github/homebrew/Library/Formula/bibtex2html.rb
Testing bibtex2html
==> /usr/local/Cellar/bibtex2html/1.98/bin/bib2bib test.bib --remove pages -ob out.bib
This is bib2bib version 1.98, compiled on Fri Nov 14 04:47:51 GMT 2014
Copyright (c) 1997-2010 Jean-Christophe Filliâtre and Claude Marché
This is free software with ABSOLUTELY NO WARRANTY (use option --warranty)

Reading test.bib...ok (1 entries).
No citation file output (no file name specified)
Error: bibtex2html: failed
test returned false
/usr/local/Library/Homebrew/debrew.rb:11:in `raise'
/usr/local/Library/Homebrew/cmd/test.rb:38:in `block (2 levels) in test'
/System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/timeout.rb:66:in `timeout'
/usr/local/Library/Homebrew/cmd/test.rb:37:in `block in test'
/usr/local/Library/Homebrew/cmd/test.rb:15:in `each'
/usr/local/Library/Homebrew/cmd/test.rb:15:in `test'
/usr/local/Library/brew.rb:135:in `<main>'
```